libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1987,6 +1986,24 @@ bool WaveTrack::CloseLock() noexcept
    return true;
 }
 
+ClipConstHolders WaveTrack::GetClipInterfaces() const
+{
+   assert(IsLeader());
```

Comment:
Again -- if I see a new assertion at the top of a function, I want to see it commented too as an explicit precondition.

The commit introducing this function repairs all the previous build problems.